templates/mod/pagespeed.conf.erb

```diff
@@ -93,6 +93,6 @@ ModPagespeedMessageBufferSize <%= @message_buffer_size %>
     SetHandler mod_pagespeed_message
 </Location>
 
-<% @additional_configuration.each_pair do |key, value| -%>
-<%= key %> <%= value %>
+<% @additional_configuration.each do |entry| -%>
+<%= entry %>
```

Comment:
This will break backwards compatibility for people who are using hashes at the moment:

```
irb(main):012:0> x = {
irb(main):013:1*
irb(main):014:1*   'ModPagespeedDisallow' => "*/jquery-ui-1.8.2.custom.min.js",
irb(main):015:1*
irb(main):016:1*   'ModPagespeedDisallow' => "*/js_tinyMCE.js",
irb(main):017:1*
irb(main):018:1* }
=> {"ModPagespeedDisallow"=>"*/js_tinyMCE.js"}
irb(main):019:0> x.collect { |entry| entry.to_s }
=> ["[\"ModPagespeedDisallow\", \"*/js_tinyMCE.js\"]"]
irb(main):020:0>
```

As you can see, this would produce weird syntax.


I've created #1276 which also has a pair of tests verifying both old and new behavior and a sentence in the README to explain the behavior.